index.html

```diff
@@ -9,22 +9,21 @@
     <!-- Bootstrap and supporting libraries -->
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css">
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
-
     <link rel="stylesheet" href="new-pipeline/style/dashboards.css">
     <script async src="https://www.googletagmanager.com/gtag/js?id=UA-49796218-2"></script>
 </head>
 <body>
-    <div class="container-fluid">
-        <header>
+        <header class="portal-header-bar">
```

Comment:
You'll want to pull `<header>` and all of its contents back to the same indentation level as `.container-fluid` to show that they're now siblings.

Reply:
okay